Test/Machine/test_machine.py

```diff
@@ -22,6 +27,30 @@ def merge_dicts(x, y):
 # Hilbert space of spins from given graph
 hi = nk.hilbert.Spin(s=0.5, graph=g)
 
+
+def randn():
+  def init(rng, shape):
+    x = jax.experimental.stax.randn()(rng, shape)
+    x = jax.numpy.asarray(x, dtype=jax.numpy.float64)
+    return x
+  return init
+
+def glorot():
+  def init(rng, shape):
+    x = jax.experimental.stax.glorot()(rng, shape)
+    x = jax.numpy.asarray(x, dtype=jax.numpy.float64)
+    return x
+  return init
+
+machines["JAX Machine"] = JAXMachine(hi,
+    jax.experimental.stax.serial(
+        jax.experimental.stax.Dense(16, glorot(), randn()),
+        jax.experimental.stax.Relu,
+        jax.experimental.stax.Dense(16, glorot(), randn()),
+        jax.experimental.stax.Relu,
+        jax.experimental.stax.Dense(2, glorot(), randn()),
```

Comment:
we should check that the output has size 2 in the non holomorphic case

Reply:
also worth checking what happens when initializing layers at random on multiple MPI cores...